docs/adr/0004-runtime-governance.md

```diff
@@ -0,0 +1,246 @@
+# ADR 0004: Runtime Governance
+
+## Changelog
+
+- 2020-09-17: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+
+Currently all runtimes can only be governed by a single entity -- the runtime
+owner. In this regard governance means being able to update certain fields in
```

Comment:
We probably need to enumerate the fields that can be updated and if update follows the same rules (range of allowed values, interaction with other values, etc) as creation.  E.g., could `GovernanceModel` be updated?